app/src/main/java/org/greenstand/android/TreeTracker/view/SelectableItemDetail.kt

```diff
@@ -30,8 +33,9 @@ import org.greenstand.android.TreeTracker.theme.CustomTheme
 
 @Composable
 fun SelectableImageDetail(
-    photoPath: String,
+    photoPath: String? = null,
     isSelected: Boolean,
+    painter: Painter? = null,
```

Comment:
Since now have multiple image types, we should avoid passing conflicting data and instead extract the image outside, and have the caller pass it in as another composable.